slick-testkit/src/main/scala/com/typesafe/slick/testkit/util/StandardTestDBs.scala

```diff
@@ -182,21 +184,16 @@ object StandardTestDBs {
     val profile = SQLServerProfile
     import profile.api.actionBasedSQLInterpolation
 
-    val defaultSchema = config.getString("defaultSchema")
+    val defaultSchema = Option(config.getStringOr("defaultSchema"))
 
-    override def localTables(implicit ec: ExecutionContext): DBIO[Vector[String]] =
-      ResultSetAction[(String,String,String, String)](_.conn.getMetaData().getTables(testDB, defaultSchema, null, null)).map { ts =>
-        ts.map(_._3).sorted
-      }
+    override def localTables(implicit ec: ExecutionContext): DBIO[Vector[String]] = {
+      MTable.getTables(None, defaultSchema, None, Some(Seq("TABLE"))).map(_.map(_.name.name))
```

Comment:
This doesn't appear to be sorted anymore. It doesn't look like we actually require that anywhere but it would make sense to drop sorting everywhere and return a `Set` to make that clear.